ocamltest/builtin_actions.ml

```diff
@@ -108,17 +108,20 @@ let not_windows = make
     "not running on Windows"
     "running on Windows")
 
-let bsd_system = "bsd_elf"
+let is_bsd_system s =
+  match s with
+  | "bsd_elf" | "netbsd" | "freebsd" | "openbsd" -> true
```

Comment:
Would it be sound just to match anything containing "bsd"?

Reply:
That was my first idea, but OCaml's pattern-matching doesn't support regexps or even substring matching, so I decided to keep it simple.